### PR TITLE
create single article page

### PR DESCRIPTION
**Pull request by sondos-ahmed**

### CODE REVIEW

File: src/api.js [hunk 1]
```diff
@@ -63,5 +63,14 @@ export function getOrderedCookingsArticles() {
 ///////// Ticket 5  View an individual article///////////
 
 export function getArticleById(article_id) {
-  return newsApp.get(`api/articles/${article_id}`);
+  return newsApp.get(`api/articles/${article_id}`).then((res) => {
+    const newArticle = { ...res.data.article };
+    const date = new Date(newArticle.created_at);
+    console.log(date);
```
Comment (yalicee) [inline src/api.js]: please remove this before you merge

File: src/api.js [hunk 2]
```diff
@@ -63,5 +63,14 @@ export function getOrderedCookingsArticles() {
...
 
 export function getArticleById(article_id) {
-  return newsApp.get(`api/articles/${article_id}`);
+  return newsApp.get(`api/articles/${article_id}`).then((res) => {
+    const newArticle = { ...res.data.article };
+    const date = new Date(newArticle.created_at);
+    console.log(date);
+    newArticle.created_at = date.toDateString();
+
+    console.log(newArticle);
```
Comment (yalicee) [inline src/api.js]: same with this

**Review by yalicee — APPROVED** [review]
Looks really good Sondos :) Happy to approve

File: src/api.js (hunk 2 above)
Comment (sondos-ahmed) [reply src/api.js]: Thank you, done

**Pull request merged**